CMakeLists.txt

```diff
@@ -635,3 +635,32 @@ if(APPLE AND TVM_IS_DEBUG_BUILD)
       VERBATIM
 		  )
 endif()
+
+#Caches the build.
+#Note that ccache-3.x doesn't support nvcc well, so CUDA kernels may never hit the cache and still
+#need to be re-compiled every time. Using ccache 4.0+ can resolve this issue.
+
+if(USE_CCACHE) # True for AUTO, ON, /path/to/ccache
+  if("${USE_CCACHE}" STREQUAL "AUTO") # Auto mode
+    find_program(CCACHE_FOUND ccache)
+    if(CCACHE_FOUND)
+      message(STATUS "Found the path to ccache, enabling ccache")
+      set(PATH_TO_CCACHE ccache)
+    else()
+      message(STATUS "Didn't find the path to CCACHE, disabling ccache")
+    endif(CCACHE_FOUND)
+  elseif("${USE_CCACHE}" STREQUAL "ON")
```

Comment:
change to use https://github.com/apache/tvm/blob/813136401a11a49d6c15e6013c34dd822a5c4ff6/cmake/utils/Utils.cmake#L86

Reply:
see https://github.com/apache/tvm/blob/813136401a11a49d6c15e6013c34dd822a5c4ff6/cmake/utils/FindEthosN.cmake#L64

Reply:
@tqchen Done!